ospd/scan.py

```diff
@@ -449,11 +449,9 @@ def simplify_exclude_host_count(self, scan_id: str) -> int:
 
         if invalid_exc_hosts > 0:
             LOGGER.warning(
-                "Please check the excluded host list. It contains "
-                "hosts which do not belong to the target. %d hosts were "
-                "removed from the excluded host list. This warning can be "
-                "ignored if this was done on purpose.",
-                invalid_exc_hosts,
+                "Please check the excluded host list. It contains hosts which "
+                "do not belong to the target. This warning can be ignored if "
+                "this was done on purpose."
```

Comment:
```suggestion
                "this was done on purpose (e.g. to exclude specific hostname)."
```

Not absolutely sure but could help to understand the message.